### ISSUE TITLE
LogWarnings in SiPixelFrameConverter:  SiPixelRawToDigi:siPixelDigis

### ISSUE THREAD

**Issue by slava77** [cms-sw/cmssw#19800]
The following warnings now appear in every event in 2017 data, for data collected starting from late July 5.
https://hypernews.cern.ch/HyperNews/CMS/get/pixelOfflineSW/1311/1/1.html

```
%MSG-w SiPixelFrameConverter:  SiPixelRawToDigi:siPixelDigis  18-Jul-2017 18:47:17 CEST Run: 299042 Event: 16902927
Map shows no fed=1202, link=9, roc=1
%MSG

```

@veszpv @boudoul if there is a TRACK JIRA ticket, please let me know so that a link can be added

**Comment by slava77** [comment]
assign reconstruction

**Comment by slava77** [comment]
+1

"fixed" in #19866 #19865 
- the warning was demoted temporarily
- to be put back to a warning once adequate false-positives detection is in place
